e2e/playwright/point-click.spec.ts

```diff
@@ -1156,188 +1156,6 @@ openSketch = startSketchOn(XY)
-          Axis: 'X',
-          AngleStart: '270',
-          Revolutions: '1',
-          Radius: '5',
-          Length: newInput,
-        },
-        commandName: 'Helix',
-      })
-      await cmdBar.submit()
-      await toolbar.closeFeatureTreePane()
-      await editor.openPane()
-      await editor.expectEditor.toContain('length = ' + newInput)
-    })
-
-    await test.step('Delete helix via feature tree selection', async () => {
-      await editor.closePane()
-      const operationButton = await toolbar.getFeatureTreeOperation('Helix', 0)
-      await operationButton.click({ button: 'left' })
-      await page.keyboard.press('Delete')
-      await scene.settled(cmdBar)
-      await editor.expectEditor.not.toContain('helix')
-      await expect(
-        await toolbar.getFeatureTreeOperation('Helix', 0)
-      ).not.toBeVisible()
-    })
-  })
-
-  test(`Helix point-and-click around segment`, async ({
-    context,
-    page,
-    homePage,
-    scene,
-    editor,
-    toolbar,
-    cmdBar,
-  }) => {
-    const initialCode = `sketch001 = startSketchOn(XZ)
-profile001 = startProfile(sketch001, at = [0, 0])
-|> yLine(length = 100)
-|> line(endAbsolute = [100, 0])
-|> line(endAbsolute = [profileStartX(%), profileStartY(%)])
-|> close()`
-    await context.addInitScript((initialCode) => {
-      localStorage.setItem('persistCode', initialCode)
-    }, initialCode)
-    await page.setBodyDimensions({ width: 1000, height: 500 })
-    await homePage.goToModelingScene()
-    await scene.settled(cmdBar)
-
-    await test.step(`Go through the command bar flow`, async () => {
-      await toolbar.closePane('code')
-      await toolbar.helixButton.click()
-      await cmdBar.expectState(initialCmdBarStateHelix)
-      await cmdBar.selectOption({ name: 'Edge' }).click()
-      await editor.selectText('yLine(length = 100)')
-      await cmdBar.progressCmdBar()
-      await page.keyboard.insertText('1')
-      await cmdBar.progressCmdBar()
-      await page.keyboard.insertText('2')
-      await cmdBar.progressCmdBar()
-      await page.keyboard.insertText('3')
-      await cmdBar.progressCmdBar()
-      await cmdBar.expectState({
-        stage: 'review',
-        headerArguments: {
-          Mode: 'Edge',
-          Edge: `1 segment`,
-          AngleStart: '2',
-          Revolutions: '1',
-          Radius: '3',
-        },
-        commandName: 'Helix',
-      })
-      await cmdBar.submit()
-      await scene.settled(cmdBar)
-    })
-
-    await test.step(`Confirm code is added to the editor, scene has changed`, async () => {
-      await toolbar.openPane('code')
-      await editor.expectEditor.toContain(
-        `
-        helix001 = helix(
-          axis = seg01,
-          radius = 3,
-          revolutions = 1,
-          angleStart = 2,
-        )`,
-        { shouldNormalise: true }
-      )
-      await toolbar.closePane('code')
-    })
-  })
-
   test(`Helix point-and-click around sweepEdge with edit and delete flows`, async ({
```

Comment:
Keeping only this one for e2e testing of the Helix flow, with all the cases moved to `geometry.test.ts`